de/manual/ch_Business.docbook

```diff
@@ -1,1279 +1,1991 @@
+            <para>Theoretisch sollen dabei der Steuerbericht und &mc.ed.tax-rep-opt; helfen, allerdings wurde die
+              deutsche Variante schon länger nicht mehr gewartet.
+            </para>
 
-                <listitem>
-                  <para><guilabel>Discount Day</guilabel> The last day of the month for the early payment discount
-                  </para>
-                </listitem>
+            <para>Eine pragmatische Vereinfachung stellt die Möglichkeit dar, die Vorsteuern als Unterkonten der
+              entsprechenden Verkaufssteuer anzulegen. In Monaten mit geringeren Verkäufen als
+              Einkäufen — Geschäftseröffnung oder Urlaub — erhalten sie dann aber negative
+              Zahlen in der Übersicht. Bevor sie diese Vereinfachung anwenden, informieren sie
+              sich, ob sie damit nicht eine verbotene Bilanzverkürzung begehen.
+            </para>
+          </glossdef>
+        </glossentry>
+      </glosslist>
 
-                <listitem>
-                  <para><guilabel>Discount %</guilabel> The discount percentage applied if paid early
-                  </para>
-                </listitem>
+      <warning>
+        <para>Wenn Sie bezüglich der steuergesetzlichen Anforderungen unsicher sind, holen Sie professionellen
+          Rat ein!
+        </para>
+      </warning>
+    </sect2>
 
-                <listitem>
-                  <para><guilabel>Cutoff Day</guilabel> The cutoff day for applying bills to the next month. After the
-                    cutoff, bills are applied to the following month. Negative values count
-                    backwards from the end of the month.
-                  </para>
-                </listitem>
-              </itemizedlist>
-            </listitem>
-          </itemizedlist>
-        </listitem>
-      </itemizedlist>
+    <sect2 id="busnss-ar-setupterms">
+      <title>Die Tabelle der Zahlungsbedingungen</title>
+
+      <abstract>
+        <para>Als Anreiz zu einer vorzeitigen Zahlung kann ein als Skonto bezeichneter Preisnachlass dienen.
+          Dieser ist mit einem kurzfristigem Zahlungsziel von Kundenrechnungen — oder
+          Lieferantenrechnungen — verbunden und kann für jede Rechnung individuell zugewiesen
+          werden.
+          <caution>
+            <para>Die Anwendung von Zahlungsbedingungen wird von &app; bisher nur teilweise unterstützt, wobei das
+              Fälligkeitsdatum korrekt ermittelt wird, allerdings wird der gewährte Nachlaß nicht
+              berechnet.
+            </para>
+          </caution>
+        </para>
+      </abstract>
+
+      <para>Die Zahlungsbedingungen werden mit Hilfe des Editors organisiert, der über den Menüeintrag
+        &mc.bus.bil-edit; aufgerufen wird. Hier haben Sie die Möglichkeit, Ihre individuellen
+        Zahlungsbedingungen und die Ihrer Lieferanten zu verwalten.
+      </para>
+
+      <figure pgwide="1">
+        <title>Zahlungsbedingungen verwalten</title>
```

Comment:
@CWehli Es läßt miir irgendwie keine Ruhe, aber hast du mal den effektiven Jahreszins von 3% Skonto errechnet?
3%/20d = 4,5%/m = 54%/a
In einem Land mit gallopierender Inflation mag das angemessen sein.
Bei uns sind meines Wissens normal:
2%/20d = 3%/m = 36%/a

Reply:
@fellen Die Vorschläge habe ich von [Wikipedia](https://de.wikipedia.org/wiki/Zahlungsbedingung#H%C3%A4ufige_Zahlungsbedingungen).
Ich bringe eine Korrektur dann im  PR #337.